activesupport/lib/active_support/cache.rb

```diff
@@ -350,6 +341,80 @@ def fetch(name, options = nil, &block)
         end
       end
 
+      # Fetches data from the cache, using the given key. If there is data in
+      # the cache with the given key, then that data is returned, otherwise +nil+
```

Comment:
```suggestion
      # the cache with the given key and that data hasn't expired, then that data is returned, otherwise +nil+
```

is that right? or will it be returned even if it's expired?

Reply:
Currently it would always be returned, but by changing the implementation to use `race_condition_ttl` it would have a slightly different behavior. I'll fix the comment with that change :+1: